common/aws/ecs/ecs.go

```diff
@@ -344,12 +348,18 @@ func (this *ECS) RegisterTaskDefinition(family string, roleARN string, networkMo
 		networkModep = aws.String(networkMode)
 	}
 
+	awsPlacementConstraints := make([]*ecs.TaskDefinitionPlacementConstraint, 0)
```

Comment:
[nit] this is more of a personal preference thing, but I've created quite a few bugs when mixing `make(object, count)` with `append`. I usually stay away from `make` unless I know I'm going to use `slice[i] = object` and not `append`.